src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -121,7 +122,7 @@ public async Task<ActionResult> SetDeleteFlagOnSystemUser(Guid systemUserId)
         /// to ensure that there is no mismatch if the same partyId creates several new SystemUsers at the same time
         /// </summary>
         /// <returns></returns>        
-        [Authorize]
+        [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_WRITE)]
```

Comment:
Also, policy needs to be added in the program.cs like this         options.AddPolicy(AuthzConstants.POLICY_SCOPE_SYSTEMREGISTER_WRITE, policy => policy
            .RequireScopeAnyOf(AuthzConstants.SCOPE_SYSTEMREGISTER_ADMIN));